RecoParticleFlow/PFRecHitProducer/plugins/alpaka/PFRecHitProducerAlpaka.cc

```diff
@@ -0,0 +1,78 @@
+#include "HeterogeneousCore/AlpakaCore/interface/alpaka/global/EDProducer.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "PFRecHitProducerKernel.h"
+#include "RecoParticleFlow/PFRecHitProducer/interface/alpaka/CalorimeterDefinitions.h"
+
+#include <utility>
+#include <vector>
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+  using namespace ParticleFlowRecHitProducer;
+
+  template <typename CAL>
+  class PFRecHitProducerAlpaka : public global::EDProducer<> {
```

Comment:
It would probably work, but I think it would be confusing, since we also have the serial implementation at: `RecoParticleFlow/PFClusterProducer/plugins/PFRecHitProducer.cc`. Then again, this file is in the `alpaka` subdirectory and the actual producers would be called `PFRecHitProducer{E,H}CAL` anyway. I do not really have a strong preference.

Reply:
An alternative could be `PFRecHitSoAProducer`, to highlight that the produced data formats are in SoA format.